### DISCUSSION TITLE
Can't import wallet core wasm in typescript/angular

### DISCUSSION THREAD

**Discussion by wolfer21** [trustwallet/wallet-core#3456] [Q&A]
Hi there!

I've been trying to use the npm library in angular and I can't seem to be able to properly import anything from the library, are perhaps the types declarations messed up?

Trying for example to import CoinType like this:
`import { CoinType } from '@trustwallet/wallet-core'`

Gives me an error saying that there is no exported CoinType in wallet-core, trying to import it from the deep path @trustwallet/wallet-core/dist/src/wallet-core will not give me any errors in the IDE but it will fail during a serve or build with the error: module @trustwallet/... not found.

**Comment by rikublock** [comment]
`CoinType`  is not a proper type. It's embedded in the webassembly (unfortunately). You need to load the core before you can use it.

```typescript
import { initWasm } from '@trustwallet/wallet-core'

const twc = await initWasm();
const { CoinType } = twc;
```
